app_src/ui/screens/camera_screen.py

```
@@ -2,345 +2,138 @@
-        Clock.schedule_once(self._reopen_camera, 0.6)
 
-    # ------------------------------------------------------------------
-    # Capture & Save (original logic unchanged)
-    # ------------------------------------------------------------------
+    def _call_lib_photo(self, permissions, grants):
+        """"""
+        print(f"permissions:{permissions}")
+        if on_android_platform():
+            if not all(grants):
+                try:
+                    print(f"QrWork::Not all permits: %s" % repr(grants))
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)